apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -328,3 +350,26 @@ type ClusterLogForwarderList struct {
 func init() {
 	SchemeBuilder.Register(&ClusterLogForwarder{}, &ClusterLogForwarderList{})
 }
+
+type LimitSpec struct {
+	// Policy is used to define the action taken
+	// by the collector when log collection rate exceeds
+	// the specified limit
+	//
+	// +kubebuilder:validation:Enum:=drop;ignore
+	// +required
+	Policy PolicyType `json:"policy,omitempty"`
+
+	// MaxRecordsPerSecond is the maximum number of log records
+	// allowed per input/output in a pipeline
+	//
+	// +required
+	MaxRecordsPerSecond resource.Quantity `json:"maxRecordsPerSecond,omitempty"`
```

Comment:
whould this be quantity?  not a long or the like?

Reply:
@alanconway Please check this comment

Reply:
@alanconway Please check this comment

Reply:
Quantity isn't wrong: "Quantity is a fixed-point representation of a number" - it doesn't _have_ to be a byte quantity. However it almost always _is_ used for byte quantities so I think it might cause confusion here since  a byte rate would also make sense. I would go with int64.